optimum/onnxruntime/configuration.py

```diff
@@ -943,12 +951,11 @@ def __init__(
         quantization: Optional[QuantizationConfig] = None,
```

Comment:
So are these type hints correct in the end? `optimization: Optional[OptimizationConfig] = None,` and `quantization: Optional[QuantizationConfig] = None,`? According to https://github.com/huggingface/optimum/pull/1089#issuecomment-1582233708 they are not?

Reply:
bump